cmd/juju/application/deploy_test.go

```diff
@@ -65,6 +66,30 @@ import (
 type DeploySuiteBase struct {
 	testing.RepoSuite
 	coretesting.CmdBlockHelper
+	charmstore jjcharmstore.CharmStorer
```

Comment:
Is the `jjcharmstore` used only for testing?

Reply:
I've changed this to the `charmstoreForDeploy` interface defined in `cmd/juju/application/deploy.go`